overlay.yml

```diff
@@ -0,0 +1,25 @@
+aliases: 
+  - name: github
+    url: https://github.com/
+
+  - name: fedorapkgs
```

Comment:
Minor: seems like if we plan to build all rpms from git repos, we don't need the fedorapkgs(as we are not fetching from it)? I happened to not include the cert and it passes. Only a minor tho :).

Reply:
The fedorapkgs is used by the distgit section but you're correct that if we had spec files in each repo we wouldn't need it.